RickLeinecker/Fortuna
# Armory change, selected tank cookie, and more.

**Description**
To start, this change will need a good review. I plan to review it with David to ensure Tank.js and TankLoader.js have been implemented well.

Complete change of armory to handle Tank class and TankLoader. Also, sets a cookie for whatever tank is currently selected and being edited in Armory.

Resolves #215 
Resolves #217 

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
1. Place a console.log(this.state.selectedTank) in changeSelectedTank function.
2. Change the selected tank.
3. You will see the previously selected tank, but it will show you how the Tank class works.

**Final Checklist:**
Go down the list and check them off.
